### PR TITLE
Add admin applications menu

### PR DESCRIPTION
**Pull request by SaraBoutej**

### CODE REVIEW

File: app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue [hunk 1]
```diff
@@ -52,7 +52,7 @@
                             class="d-flex flex justify-center mx-2">
                         <a
                                 class="primary--text seeAllApplicationsBtn"
-                                href="/portal/dw/appCenterUserSetup"
+                                :href="appCenterUserLink"
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue]:
```suggestion
                                :href="appCenterUserSetupLink"
```

File: app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue [hunk 2]
```diff
@@ -66,14 +66,16 @@
         data () {
             return {
                 appLauncherDrawer : null,
-                favoriteApplicationsList: []
+                favoriteApplicationsList: [],
+                appCenterUserLink: ''
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue]:
```suggestion
                appCenterUserSetupLink: ''
```

File: app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue [hunk 3]
```diff
@@ -66,14 +66,16 @@
...
             }
         },
         created() {
             this.getFavoriteApplicationsList();
             window.require(['SHARED/appCenterBundle'], function (appCenterBundle) {
                 appCenterBundle.init();
             });
+            this.appCenterUserLink = eXo.env.portal.context + "/" + eXo.env.portal.portalName + "/appCenterUserSetup";
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue]:
```suggestion
            this.appCenterUserSetupLink= eXo.env.portal.context + "/" + eXo.env.portal.portalName + "/appCenterUserSetup";
```

File: app-center-webapps/src/main/webapp/vue-apps/myApplications/components/MyApplications.vue [hunk 4]
```diff
@@ -15,7 +15,7 @@
 			</li>
 		</ul>
 		<div v-if="maxFavoriteApps == undefined || favoriteApplicationsList.length < maxFavoriteApps" class="addTool">
-      		<a href="/portal/intranet/appCenterUserSetup">
+      		<a :href="appCenterUserLink">
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/vue-apps/myApplications/components/MyApplications.vue]:
```suggestion
      		<a :href="appCenterUserSetupLink">
```

File: app-center-webapps/src/main/webapp/vue-apps/myApplications/components/MyApplications.vue [hunk 5]
```diff
@@ -34,13 +34,15 @@
         data(){
             return{
             	favoriteApplicationsList: [],
-            	maxFavoriteApps: ''
+            	maxFavoriteApps: '',
+				appCenterUserLink: ''
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/vue-apps/myApplications/components/MyApplications.vue]:
```suggestion
				appCenterUserSetupLink: ''
```

File: app-center-webapps/src/main/webapp/vue-apps/myApplications/components/MyApplications.vue [hunk 6]
```diff
@@ -34,13 +34,15 @@
...
             }
         },
 
         created() {
         	this.getFavoriteApplicationsList();
         	this.getMaxFavoriteApps();
+			this.appCenterUserLink = eXo.env.portal.context + "/" + eXo.env.portal.portalName + "/appCenterUserSetup";
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/vue-apps/myApplications/components/MyApplications.vue]:
```suggestion
			this.appCenterUserSetupLink= eXo.env.portal.context + "/" + eXo.env.portal.portalName + "/appCenterUserSetup";
```

File: app-center-webapps/src/main/webapp/skin/main.css [hunk 7]
```diff
@@ -820,7 +820,7 @@
 }
 
 .uiApplicationIcon {
-  color: white!important;
+  color: #a8b3c5!important;
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/skin/main.css]: remove this modification

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: app-center-webapps/src/main/webapp/skin/main.css (hunk 7 above)
Comment (SaraBoutej) [reply app-center-webapps/src/main/webapp/skin/main.css]: Done!

File: app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue (hunk 1 above)
Comment (SaraBoutej) [reply app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue]: Done!

File: app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue (hunk 2 above)
Comment (SaraBoutej) [reply app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue]: Done!

File: app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue (hunk 3 above)
Comment (SaraBoutej) [reply app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue]: Done!

File: app-center-webapps/src/main/webapp/vue-apps/myApplications/components/MyApplications.vue (hunk 4 above)
Comment (SaraBoutej) [reply app-center-webapps/src/main/webapp/vue-apps/myApplications/components/MyApplications.vue]: Done!

File: app-center-webapps/src/main/webapp/vue-apps/myApplications/components/MyApplications.vue (hunk 5 above)
Comment (SaraBoutej) [reply app-center-webapps/src/main/webapp/vue-apps/myApplications/components/MyApplications.vue]: Done!

File: app-center-webapps/src/main/webapp/vue-apps/myApplications/components/MyApplications.vue (hunk 6 above)
Comment (SaraBoutej) [reply app-center-webapps/src/main/webapp/vue-apps/myApplications/components/MyApplications.vue]: Done!

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**